vue-components/src/components/OptionsMenu.vue

```diff
@@ -77,6 +77,13 @@ export default Vue.extend( {
 			type: Number,
 			default: -1,
 		},
+		/**
+		 * scroll lookup menu items into view while naviagitng using keyboard
+		 */
+		scrollWithKeyboard: {
```

Comment:
I'm not sure there will ever be a situation where we would not want to scroll with the keyboard. Which circumstances do you have in mind for this?

Reply:
ah that was something i forgot to remove. i was thinking about the dropdown menu. but we do need it in that case as well